CMakeLists.txt

```diff
@@ -943,6 +943,19 @@ if(CURL_USE_RUSTLS)
     string(APPEND CMAKE_C_FLAGS " ${RUSTLS_CFLAGS}")
   endif()
 
+  if(RUSTLS_VERSION AND RUSTLS_VERSION VERSION_GREATER_EQUAL "0.15")
+    set(RUSTLS_HAS_HPKE TRUE)
```

Comment:
How about the name `HAVE_RUSTLS_SUPPORTED_HPKE` to align with the rest of the code?

Reply:
SGTM. Switched.